host-controller/src/main/java/org/jboss/as/domain/controller/operations/DomainServerLifecycleHandlers.java

```diff
@@ -439,6 +444,20 @@ public static void registerSuspendedStartTransformers(ResourceTransformationDesc
 
     }
 
+    // older releases don't have all of the host-release enum values, so we filter them
+    // the host-release values are sent via attribute on an add of host-exclude
+    // { "operation" => "add",
+    //         "address" => [("host-exclude" => "EAP70")],
+    //          "host-release" => "EAP7.0",
+    //  ... }
+    // TODO: WFCORE-3252 make this filtering depend on the slaves version and only send that and prior known values?
+    public static void registerHostReleaseForPriorReleaseEnumValuesTransformers(ResourceTransformationDescriptionBuilder builder) {
+        builder.addOperationTransformationOverride(ADD)
+                .setDiscard(new DiscardAttributeChecker.DiscardAttributeValueChecker(new ModelNode(HOST_RELEASE)), EAP70)
+                .addRejectCheck(RejectAttributeChecker.DEFINED, EAP70)
+                .end();
+    }
```

Comment:
This is unused and I don't think it's needed.

Reply:
Hmm, so you mean just leave it with the .addRejectCheck, but drop the .setDiscard? Ah, yes, that would make sense, since it would never be understood if it were present.

Reply:
Sorry; I missed where this is used in the other class; I thought the whole method was cruft and was saying it should be removed.

Reply:
Never mind, its just a missing registration for 4.1 on the existing host-exclude, thanks for correcting me :)